packages/amplify-category-auth/resources/auth-custom-resource/hostedUILambda.js

```diff
@@ -18,7 +18,7 @@ exports.handler = (event, context) => {
     .catch((err) => {
       console.log(err);
 
-      if (err.name === 'NotFoundException') {
+      if (err.name === 'NotFoundException' || err.message === 'No such domain or user pool exists.') {
```

Comment:
How does `err` look like when `No such domain or user pool exists.` is thrown?
Is there any other property like `code` that could be used instead of message in this condition?
I'm not sure if using message here is sustainable solution, not sure if service considers changing messaging a breaking change.

Reply:
By looking at this https://stackoverflow.com/questions/54002030/how-to-fix-cognito-user-pool-domain-destruction-with-terraform-so-the-user-pool I seems that error should contain input values that were wrong, we could use that instead of the message.

Reply:
I can add `err.name === 'InvalidParameterException'`